### PR TITLE
Update mkdocs

### PR DESCRIPTION
**Pull request by KasunAratthanage**
- Amazon DynamoDB Connector
- BigQuery Connector
- Google Pub Sub Connector
- Redis Connector

### CODE REVIEW

File: en/micro-integrator/mkdocs.yml [hunk 1]
```diff
@@ -522,6 +522,19 @@ nav:
           - 'SMPP Reference': 'references/connectors/smpp-connector/smpp-connector-config.md'
           - 'SMPP Inbound Endpoint Example': 'references/connectors/smpp-connector/smpp-inbound-endpoint-example.md'
           - 'SMPP Inbound Endpoint Reference': 'references/connectors/smpp-connector/smpp-inbound-endpoint-config.md'
+        - Amazon DynamoDB Connector:
```
Comment (samgnaniah) [inline en/micro-integrator/mkdocs.yml:525]: I think this is already added. Please check.

File: en/micro-integrator/mkdocs.yml [hunk 2]
```diff
@@ -522,6 +522,19 @@ nav:
...
+          - 'Amazon DynamoDB Connector Example': 'references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md'
+        - BigQuery Connector:
+          - 'Setting up the BigQuery Environment': 'references/connectors/bigquery-connector/bigquery-connector-configuration.md'
+          - 'BigQuery Connector Example': 'references/connectors/bigquery-connector/bigquery-connector-example.md'
+        - Google Pub Sub Connector:
+          - 'Setting up the Google Pub Sub Environment': 'references/connectors/google-pubsub-connector/googlepubsub-connector-configuration.md'
+          - 'Google Pub Sub Connector Example': 'references/connectors/google-pubsub-connector/googlepubsub-connector-example.md'
+        - Redis Connector:
+          - 'Setting up the Environment': 'references/connectors/redis-connector/redis-connector-configuration.md'
+          - 'Redis Connector Example': 'references/connectors/redis-connector/redis-connector-example.md'
```
Comment (samgnaniah) [inline en/micro-integrator/mkdocs.yml:537]: Let's add these in alphabetic order.

File: en/micro-integrator/mkdocs.yml (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/mkdocs.yml:525]: Yes already added.Updated.

File: en/micro-integrator/mkdocs.yml (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/mkdocs.yml:537]: Updated.

**Pull request merged**